src/Oci8/Oci8Connection.php

```diff
@@ -2,23 +2,27 @@
 
 namespace Yajra\Oci8;
 
-use PDO;
-use PDOStatement;
-use Yajra\Pdo\Oci8\Statement;
-use Yajra\Oci8\Schema\Trigger;
-use Yajra\Oci8\Schema\Sequence;
-use Illuminate\Database\Grammar;
-use Illuminate\Database\Connection;
 use Doctrine\DBAL\Connection as DoctrineConnection;
-use Yajra\Oci8\Query\OracleBuilder as QueryBuilder;
-use Yajra\Oci8\Schema\OracleBuilder as SchemaBuilder;
 use Doctrine\DBAL\Driver\OCI8\Driver as DoctrineDriver;
+use Exception;
+use Illuminate\Database\Connection;
+use Illuminate\Database\Grammar;
+use Illuminate\Support\Str;
+use PDO;
+use PDOStatement;
 use Yajra\Oci8\Query\Grammars\OracleGrammar as QueryGrammar;
+use Yajra\Oci8\Query\OracleBuilder as QueryBuilder;
 use Yajra\Oci8\Query\Processors\OracleProcessor as Processor;
 use Yajra\Oci8\Schema\Grammars\OracleGrammar as SchemaGrammar;
+use Yajra\Oci8\Schema\OracleBuilder as SchemaBuilder;
+use Yajra\Oci8\Schema\Sequence;
+use Yajra\Oci8\Schema\Trigger;
+use Yajra\Pdo\Oci8\Statement;
 
```

Comment:
I think the cs for laravel import is it was arranged from shortest to longest name. Might be the reason why styleci is failing.